### PR TITLE
jan 4x4

### PR DESCRIPTION
**Pull request by BnjmnZmmrmn**
**contents**
- add build rules for jan in makefile
- mjan game implementation

**reviewers**
i discarded verbose docs, lmk if i should revert to template. please nit

### CODE REVIEW

**Comment by BnjmnZmmrmn** [comment]
fixing branch

File: src/mjan.c [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+	  board[9],
+	  board[10],
+	  board[11]
+  );
+
+  printf("         ( 13 14 15 16 )           : %c %c %c %c %s\n\n", // row four
+	  board[12],
+	  board[13],
+	  board[14],
+    board[15],
```
Comment (jparkk04) [inline src/mjan.c:238]: should there be a tab here or is it intentionally no tab?

**Review by jparkk04 — APPROVED** [review]
The code's logic is good and it is written nicely

File: src/mjan.c (hunk 1 above)
Comment (BnjmnZmmrmn) [reply src/mjan.c:238]: wow good find actually, will fix in  next commit

**Pull request merged**